docs/releases/2.0.txt

```diff
@@ -281,6 +281,8 @@ Miscellaneous
 * The default HTTP error handlers (``handler404``, etc.) are now callables
   instead of dotted Python path strings. Django favors callable references
   since they provide better performance and debugging experience.
+* Management command raises ``TypeError``, if an unrecognized argument is
```

Comment:
This note should be more prominent (not under Miscellaneous) and explain how to use `private_options` which should be documented where the other management command attributes are.